src/maps/deployment.ts

```diff
@@ -112,3 +116,44 @@ function mapDeploymentConcurrencyOptionsToDeploymentApiConcurrencyOptions(source
     collision_strategy: source.collisionStrategy,
   }
 }
+
+
+export const mapDeploymentVersionResponseToDeploymentVersion: MapFunction<DeploymentVersionResponse, DeploymentVersion> = function(source) {
+  return new DeploymentVersion({
+    id: source.id,
+    created: this.map('string', source.created, 'Date'),
+    createdBy: this.map('CreatedOrUpdatedByResponse', source.created_by, 'CreatedOrUpdatedBy'),
+    updated: this.map('string', source.updated, 'Date'),
+    updatedBy: this.map('CreatedOrUpdatedByResponse', source.updated_by, 'CreatedOrUpdatedBy'),
+    name: source.name,
+    versionInfo: this.map('DeploymentVersionInfoResponse', source.version_info, 'DeploymentVersionInfo'),
+    description: source.description,
+    flowId: source.flow_id,
+    deploymentId: source.deployment_id,
+    paused: source.paused,
+    parameters: source.parameters,
+    parameterOpenApiSchema: schemaV2Mapper.map('SchemaResponse', source.parameter_openapi_schema ?? {}, 'Schema'),
+    tags: source.tags ? sortStringArray(source.tags) : null,
+    manifestPath: source.manifest_path,
+    path: source.path,
+    entrypoint: source.entrypoint,
+    storageDocumentId: source.storage_document_id,
+    infrastructureDocumentId: source.infrastructure_document_id,
+    jobVariables: source.job_variables,
+    workQueueName: source.work_queue_name,
+    workPoolName: source.work_pool_name,
+    enforceParameterSchema: source.enforce_parameter_schema,
+    pullSteps: source.pull_steps,
+    can: createObjectLevelCan(),
+    status: this.map('ServerDeploymentStatus', source.status, 'DeploymentStatus'),
+    disabled: source.disabled ?? false,
+    globalConcurrencyLimit: this.map('ConcurrencyV2LimitResponse', source.global_concurrency_limit, 'ConcurrencyV2Limit'),
+    concurrencyOptions: source.concurrency_options == null ? null : mapDeploymentApiConcurrencyOptionsToDeploymentConcurrencyOptions(source.concurrency_options),
+  })
+}
+
+export const mapDeploymentVersionInfoResponseToDeploymentVersionInfo: MapFunction<DeploymentVersionInfoResponse, DeploymentVersionInfo> = function(source) {
+  return Object.fromEntries(
+    Object.entries(source).map(([key, value]) => [camelCase(key), value]),
+  ) as DeploymentVersionInfo
```

Comment:
flagging this - not super keen on this but also not sure what all this blob will contain